### PR TITLE
Is is faster than ==

### PR DESCRIPTION
**Pull request by PendulumDreams**

### CODE REVIEW

**Comment by PendulumDreams** [comment]
This didn't do what I wanted.

**Comment by PendulumDreams** [comment]
I want to merge all my files into a single commit first.

File: direct/src/cluster/ClusterClient.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def controlObjectTask(self, task):
 
     def sendNamedMovementDone(self, serverList = None):
 
-        if (serverList == None):
+        if (serverList is None):
```
Comment (rdb) [inline direct/src/cluster/ClusterClient.py]: While you're at it making style changes, perhaps you could also remove the unnecessary parentheses in `if` statements like this one.

**Comment by rdb** [comment]
For future reference, GitHub can automatically squash multiple commits when merging a pull request.

**Pull request closed without merging**